### PR TITLE
chore: clean sonar lint issues

### PR DESCRIPTION
**Pull request by Goldziher**
This PR cleans sonar lint issues

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -63,70 +65,71 @@
...
-    ("py:class", "_orm.registry"),
-    ("py:class", "_schema.MetaData"),
-    ("py:class", "_schema.Table"),
-    ("py:class", "_RegistryType"),
-    ("py:class", "abc.Collection"),
-    ("py:class", "TypeEngine"),
-    ("py:class", "ExternalType"),
-    ("py:class", "UserDefinedType"),
-    ("py:class", "_types.TypeDecorator"),
+    (PY_CLASS, "BaseModel"),
```
Comment (peterschutt) [inline docs/conf.py:68]: I understand why, but I don't think this really improves anything, as these directives aren't ever going to change.

If we are to go through with it,  `py:meth` and `py:obj` should be treated the same for sake of consistency.

**Review by peterschutt — APPROVED** [review]

File: docs/conf.py (hunk 1 above)
Comment (Goldziher) [reply docs/conf.py:68]: its about repetation of values - sonar doesnt like copy pasting.

Comment (peterschutt) [reply docs/conf.py:68]: Yes, as I said, I understand why, but I don't think it adds anything. Not a blocker though, although I do think the consistency point is pertinent.

Comment (Goldziher) [reply docs/conf.py:70]: addressed

**Pull request merged**